### ISSUE TITLE
Launcher is randomly unset as default

### ISSUE THREAD

**Issue by Thv1337** [SimpleMobileTools/Simple-Launcher#59]
Android 13. Pixel 7. Latest github release. For no apparent reason it asks me to choose the default launcher.

**Comment by tibbi** [comment]
do other launcher apps work better?

**Comment by Thv1337** [comment]
Yes, I have tried with Olauncher, for example, and this does not happen to me.

**Comment by Thv1337** [comment]
It just happened to me when I created a shortcut of the page on the desktop with the browser

**Comment by tibbi** [comment]
so you can reproduce it reliably like that? Default works just fine, but breaks after creating a shortcut?

**Comment by Thv1337** [comment]
In another circumstance it has also happened to me, when adding a widget I would say that too.

**Comment by Thv1337** [comment]
In another circumstance it has also happened to me, when adding a widget I would say that too.

**Comment by Cwpute** [comment]
@sandoxy described a common crashing-default-unsetting situation in their issue #154 before marking it as duplicate of this one.

So it seems Launcher unsets itself as the default launcher everytime it crashes. I can personaly confirm this. It musn't be from adding a widget, it can also come from browsing the drawers too fast etc…

**Comment by inson1** [comment]
@Cwpute Do you know about the sale and the fork, right?
